gcloud-java-vision/src/test/java/com/google/cloud/vision/spi/v1/ImageAnnotatorTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.vision.spi.v1;
+
+import com.google.api.gax.core.PageAccessor;
```

Comment:
Right now all these import are statically determined so depending on the tests some of them may be unused. We could either dynamically determine imports during generation or add a post-processor to remove unused ones.